### PR TITLE
Update SC-7(4) for RHCOS and OCP4

### PR DESCRIPTION
**Pull request by redhatrises**

### CODE REVIEW

**Comment by redhatrises** [comment]
@JAORMX for your review

File: openshift-container-platform-4/policies/SC-Systems_and_Communications_Protection/component.yaml [hunk 1]
```diff
@@ -278,39 +278,33 @@
...
       text: |
-        This control is inherited from the network and IaaS provider.
-        To assist with this organizational control, documentation is being
-        planned. Progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-425
+        Protecting the confidentiality and integrity of the information being transmitted
+        across each interface is an organizational control outside the
+        scope of OpenShift configuration and is typically handled at the network
+        infrastructure layer.
```
Comment (JAORMX) [inline openshift-container-platform-4/policies/SC-Systems_and_Communications_Protection/component.yaml:295]: Could we also mention here that for the control plane components in OpenShift we do use mutual TLS? Or is that not in scope in this doc?

**Review by JAORMX — APPROVED** [review]

File: openshift-container-platform-4/policies/SC-Systems_and_Communications_Protection/component.yaml (hunk 1 above)
Comment (redhatrises) [reply openshift-container-platform-4/policies/SC-Systems_and_Communications_Protection/component.yaml:295]: @JAORMX this control is about external telecommunication service(s). Per our conversation, OCP shouldn't or does not interface with external telecommunication service(s) which is typically done at the switch or router layer. Do you disagree?

Comment (JAORMX) [reply openshift-container-platform-4/policies/SC-Systems_and_Communications_Protection/component.yaml:295]: Oh, you're right. I got confused there. Then this is good.

**Comment by JAORMX** [comment]
LGTM
